api/src/main/java/org/openmrs/module/fhir2/FhirConstants.java

```diff
@@ -28,5 +28,7 @@ private FhirConstants() {
 	public static String OPENMRS_FHIR_EXT_NAME = OPENMRS_FHIR_EXT_PREFIX + "/name";
 	
```

Comment:
If these are constants, is there a reason why they are not declared as `final`?

Reply:
This is my fault. The initial "constants" I created were using Spring property injection a la `OPENMRS_FHIR_SERVER_VERSION`, which cannot be declared final. I'll patch these up in a separate PR.

Reply:
This is my fault. The initial constants in this module where all set via Spring property injection, as we're doing for `OPENMRS_FHIR_SERVER_VERSION` and such "constants" can't be declared final for obvious reasons. I'll be patching these up in a separate PR.